_posts/de/2020-10-02-ALTF4-7.md

```diff
@@ -1,87 +1,88 @@
+
+Yuoki Industries durchlief kürzlich harte Veränderungen, da der Modentwickler YuokiTani (auch bekannt dafür, für viele andere Factoriomods Grafiken zu liefern) keine Zeit mehr hat, um selbst Factorio zu spielen und so nicht mehr in der Lage ist, neue Inhalte zu testen.
+
+Um dieser Situation Abhilfe zu schaffen, begründete YuokiTani ein [GitHub-Repository](https://github.com/Yuoki-Industries) für seine Mods, wodurch nun Entwickler aus der Community ihn unterstützen können. Dadurch wurde der Hauptmod überarbeitet und auf den aktuellen Stand der Technik und auf ein Level mit dem Grundspiel gebracht.
+
+Die Mod wurde so in den folgenden Wegen verbessert:
+
+1. Es gibt nun Einstellungen:
+    * Abschalten des Verhaltens der Vanilla-Produktivitätsmodule
+    * Einstellbare Spielerreichweite und Inventargröße
+    * Möglichkeit zum Schnellstart (mit Yuoki Powerarmour Stufe 1, einem Schild, Batterie, Generator, Roboterhangar und 50 Baurobotern)
+2. Abbauzeiten wurden verkürzt um an das Grundspiel angeglichen zu werden
+3. Pufferkisten wurden hinzugefügt
+4. Lagerkisten haben nun Logistikfilter
+5. Alle Yuoki-Radare werden jetzt auf der Karte dargestellt
+6. Yuoki-Erzförderer können mit dem Schaltungsnetz verbunden werden
+
+Auch [Yuoki Industries Engines](https://mods.factorio.com/mods/YuokiTani/yi_engines) wurde an vielen Stellen verbessert, ist aber trotzdem noch nicht auf dem selben Niveau wie die Hauptmod. In diesem Prozess werden auch die anderen Yuoki-Industries-Mods je nach Bedarf überarbeitet und wenn nötig verbessert.
+
+Für alle Interessierten, die Yuoki Industries in Aktion sehen möchten: Ich werde ab dem dritten Oktober jeweils Montags und Samstags um 10:00 UTC+1 auf [Twitch](https://www.twitch.tv/churchorganist) eine neue Fabrik streamen (englischsprachig). Währenddessen beantworte ich gerne Fragen zu den Mods.
+
+Viel Spaß beim Bauen!
+
+## Megabasis Denkweise <author>LoneWolf</author>
+
+In der [vierten Ausgabe von Alt-F4](https://alt-f4.blog/de/ALTF4-4/#designing-blueprints-with-creative-mod-and-kirkmcdonalds-calculator-gamebuster800) hat gamebuster800 kurz die Idee angesprochen, Erz direkt bei den Erzförderern zu schmelzen, statt an einem zentralen Ort. Der Grund dafür ist ziemlich unkompliziert und einfach zu verstehen: Erz stapelt sich auf 50, während Platten eine Stapelgröße von 100 erreichen. Daher bewegt man nur halb so viel, wenn man eine Ladung Erz transportiert, im Vergleich zu einer Ladung Platten. In kleinerem Maße, zum Beispiel wenn man seine Fabrik gerade erst begonnen hat, ist das kein Problem, aber wenn man in dem Bereich von zehntausenden Erz und Platten pro Minute kommt, kann so etwas einen großen Unterschied machen. Sogar mit Fließbändern ist Platten transportieren effizienter als Erz, aber mit Zügen ist der Unterschied überwältigend.

```

Comment:
Alt-F4-4 Anchor Link

Reply:
#designing-blueprints-with-creative-mod-and-kirkmcdonalds-calculator-gamebuster800

Reply:
#das-entwickeln-von-blaupausen-mit-creative-mod-und-kirkmcdonalds-rechner-gamebuster800